File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/PatchBillingCreditCardVerifyRequest.scala

package org.openapitools.client.model


case class PatchBillingCreditCardVerifyRequest (
    /* The CVV/CVC code on the back of the credit card. */
    _cc_ccv2: String
)
object PatchBillingCreditCardVerifyRequest {
    def toStringBody(var_cc_ccv2: Object) =
        s"""
        | {
        | "cc_ccv2":$var_cc_ccv2
        | }
        """.stripMargin
}

File: OPENAPI-REST-API/openapi-client/zapier/models/patchBillingCreditCardVerify_request.js
const utils = require('../utils/utils');

module.exports = {
    fields: (prefix = '', isInput = true, isArrayChild = false) => {
        const {keyPrefix, labelPrefix} = utils.buildKeyAndLabel(prefix, isInput, isArrayChild)
        return [
            {
                key: `${keyPrefix}cc_ccv2`,
                label: `The CVV/CVC code on the back of the credit card. - [${labelPrefix}cc_ccv2]`,
                required: true,
                type: 'string',
            },
        ]
    },
    mapping: (bundle, prefix = '') => {
        const {keyPrefix} = utils.buildKeyAndLabel(prefix)
        return {
            'cc_ccv2': bundle.inputData?.[`${keyPrefix}cc_ccv2`],
        }
    },
}
